.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__/
 
 # Distribution / packaging
 .Python
+migrations/
```

Comment:
Remove this line and commit your migrations.
For more info check: https://stackoverflow.com/a/28035246/15390079

Reply:
Noted, thank you